root/components/custom-searchbar.js

```diff
@@ -238,10 +273,15 @@ class Searchbar extends HTMLElement {
     const checkboxContainer = createCheckboxContainer();
     searchbarContainer.appendChild(checkboxContainer);
 
+    form.appendChild(ingInputi);
+    form.appendChild(ingInpute);
```

Comment:
Quick note, maybe visually it would be better to put these two text inputs under a [collapsible element](https://www.w3schools.com/howto/howto_js_collapsible.asp), a dropdown called "Ingredients -" would suffice. That way the user isn't overwhelmed with 3 text inputs for the search page.